test/e2e/page-objects/flows/transaction.ts

```diff
@@ -38,3 +38,35 @@ export const createDappTransaction = async (
     },
   ]);
 };
+
+/**
+ * This function creates the steps required to send a transaction from the homepage to final confirmation without confirm.
+ *
+ * @param params - An object containing the parameters.
+ * @param params.driver - The webdriver instance.
+ * @param params.recipientAddress - The recipient address.
+ * @param params.amount - The amount of the asset to be sent in the transaction.
+ */
+export const createTransactionToAddress = async ({
```

Comment:
Is this not the same as `createInternalTransaction` above? Could we just add optional arguments to that for a custom recipient or amount?